kahuna/public/js/components/gr-top-bar/gr-top-bar.css

```diff
@@ -2,7 +2,7 @@ gr-top-bar {
     display: block;
     height: 50px; /* faking we're in the flow */
     position: relative;
-    z-index: 3;
+    z-index: 30;
```

Comment:
Any rationale for this change? Will need to check how that fits with other overlays etc.